### ISSUE TITLE
[Bug] AMOUNT_OF_BUY_TX_BEFORE_THE_BOT_STOP affecting sell

### ISSUE THREAD

**Issue by AdamB2023** [tsarbuig/WalletHunter#58]
This was sell TX and WH didn't sold due to buy limit.
```
-- SWAP DETECTED -----------------------------------------------
02-20 12:21:05.215  Tx 0x38d66f467e276732d10b6c727c5de1603a517c4cc5ce87694c681ccb1fc048fc from #62: 0x5758C8Ea1Ffd8Cfc2100e74a47e91Fa8e46027F6 - id: 48fc
02-20 12:21:05.215  48fc - SHOULD WE CONTINUE? -- BUY Tx made in a row = 3 | AMOUNT_OF_BUY_TX_BEFORE_THE_BOT_STOP parameter = 3 --> cancel BUY
```

**Comment by tsarbuig** [comment]
this is a "log only" error, bot SHOULD still have sold token. Are you sure it did not ?
Anyway, just tested and solved this log error in v0.50.2b

**Comment by AdamB2023** [comment]
You are right, it's ok, I missed another log.
WH didn't sell in fact due to `SELL_TX_MINIMUM_EARNINGS_IN_DOLLARS `
